example/config/app.conf.json

```diff
@@ -98,11 +98,82 @@
           "membership": {"subsystem": "metering"},
           "appearance": {"title": "PV Array"},
           "location": {"title": "Roof", "floor": "Roof"}
+        },
+        {
+          "name": "smart-core/meters/03",
+          "traits": [{"name": "smartcore.bos.Meter", "more": {"unit":  "litres"}}],
+          "membership": {"subsystem": "metering"},
+          "appearance": {"title": "Water Meter"},
+          "location":   {"title": "Ground Floor", "floor":"Ground"}
         }
       ]
     }
   ],
   "automation": [
+    {
+      "name": "smart-core/meters/01/meter-history",
+      "source": {
+        "name": "smart-core/meters/01",
+        "trait": "smartcore.bos.Meter"
+      },
+      "storage": {
+        "type": "postgres",
+        "uri": "postgres://postgres@localhost:5432/smart_core",
+        "passwordFile": ".data/secrets/postgres-password"
+      },
+      "type": "history"
+    },
+    {
+      "name": "smart-core/meters/03/meter-history",
+      "source": {
+        "name": "smart-core/meters/03",
+        "trait": "smartcore.bos.Meter"
+      },
+      "storage": {
+        "type": "postgres",
+        "uri": "postgres://postgres@localhost:5432/smart_core",
+        "passwordFile": ".data/secrets/postgres-password"
+      },
+      "type": "history"
+    },
+    {
+      "name":     "wordpress-test",
+      "type":     "wordpress",
+      "disabled": true,
+      "base_url": "https://vanti-plugin-test-com.stackstaging.com/wp-json/recording/v1/create",
+      "site":     "abc-test1",
+      "authentication": {
+        "type":       "Authorization Bearer",
+        "secretFile": "./.data/secrets/wordpress"
+      },
+      "sources": {
+        "occupancy":    {
+          "path": "occupancy",
+          "interval": "10s",
+          "sensors":  ["pir/01","pir/02","pir/03"]
+        },
+        "temperature": {
+          "path": "temperature",
+          "interval": "11s",
+          "sensors": ["FCU/01","FCU/02"]
+        },
+        "energy":       {
+          "path": "energy",
+          "interval": "1m",
+          "meters": ["smart-core/meters/01"]
+        },
+        "air_quality":  {
+          "path": "air_quality",
+          "interval": "10s",
+          "sensors": ["smart-core/iaq/01"]
+        },
+        "water": {
+          "path": "water",
+          "interval": "10s",
+          "meters" : ["smart-core/meters/03"]
+        }
+      }
+    },
```

Comment:
I don't think this test belongs in the core example config. Have a look at some of the email autos, they have custom testing tools written to do this kind of thing

Reply:
I have updated with a tool package for the wordpress automation